dashboard/package.json

```diff
@@ -84,14 +84,14 @@
     "webpack": "^1.12.11",
     "webpack-stream": "~2.1.0",
     "wrench": "~1.5.8",
-    "yarn": ">=1.2.1"
+    "yarn": "^1.13.0"
```

Comment:
how is this change related to "fix build output for UD"?

Reply:
I need to upgrade the yarn for using option.

Try to be shorter as possible:  you can try to build this branch with docker
Use this
```
RUN cd dashboard && yarn --ignore-optional
```
instead of
```
RUN cd dashboard && npm i yarn && ./node_modules/.bin/yarn yarn --ignore-optional
```

And you will have 'BUILD FAILURE' (option support with old yarn version)

Reply:
I have removed "yarn" from the devDependencies at all.